features/step_definitions/common_steps.rb

```diff
@@ -404,26 +404,38 @@
   end
 end
 
-Then /^I (don't |)see '(.*)' text on the page/ do |negative, expected_text|
+Then /^I (don't |)see '([^\']*)' text on the page/ do |negative, expected_text|
   has_content = page.has_content?(expected_text)
+
   if negative.empty?
     expect(has_content).to be true
   else
     expect(has_content).to be false
   end
 end
 
+Then /^I (don't |)see '(.*)' or '(.*)' text on the page/ do |negative, expected_text, alternative_expected_text|
+  has_content = page.has_content?(expected_text)
+  has_alternative_content = page.has_content?(alternative_expected_text)
+
+  if negative.empty?
+    expect(has_content || has_alternative_content).to be true
+  else
+    expect(has_content || has_alternative_content).to be false
+  end
+end
+
 Then /^I see a '(.*)' attribute with the value '(.*)'/ do |attribute_name, attribute_value|
   place = "//*[@#{attribute_name} = \"#{attribute_value}\"]"
   expect(all(:xpath, place).length).to eq(1)
 end
 
 Then /^I take a screenshot/ do
-  page.save_screenshot('screenshot.png')
+  page.save_screenshot('screenshot.png', full: true)
```

Comment:
What does the "full" screenshot give?

Reply:
It screenshots the entire page, not just the part of the page currently visible  in the 'browser' window.

Reply:
a full screenshot, instead of what fits on capybara screen